cmd/swarm/main.go

```diff
@@ -80,6 +80,7 @@ var gitCommit string
 var (
 	SwarmErrNoBZZAccount        = "bzzaccount option is required but not set; check your config file, command line or environment variables"
 	SwarmErrSwapSetNoBackendURL = "SWAP is enabled but --swap-backend-url is not set"
+	SwarmErrSwapNotSetLogPath   = "SWAP is disable but --swap-audit-logpath is set; check your config file and enable swap"
```

Comment:
Small typo here. SWAP is disable**d**. 
There are two options to resolve:
- disable swap (note: can be done also via command line or env variable, not only config)
- disable --swap-audit-logpath

Also, I am not so sure if it is worthy to stop the node from booting up if the node provides the --swap-audit-logpath but not swap by itself. What would be a good solution as well, is just to disregard this flag if SWAP is not set.

Reply:
Fixed.
I removed the error and uses the default logger in case the logger path is not set.
So the normal boot sequence can continue.